### PR TITLE
Add a flag to ignore custom domain configurations when building service urls

### PR DESCRIPTION
**Pull request by Yasasr1**
### Proposed changes in this pull request
Added a flag that can be used to skip custom domains when building service urls. When this is set to true, custom domain configurations for the tenant will be ignored and the service urls will be built using the default builder

### CODE REVIEW

File: components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java [hunk 1]
```diff
@@ -58,6 +58,7 @@ public class DefaultServiceURLBuilder implements ServiceURLBuilder {
     protected boolean mandateTenantedPath = false;
     protected Map<String, String> parameters = new HashMap<>();
     protected Map<String, String> fragmentParams = new HashMap<>();
+    protected boolean skipCustomDomain = false;
```
Comment (mevan-karu) [inline components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java:61]: `skipDomainBranding` WDYT?

Comment (mevan-karu) [reply components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java:61]: Also let's add a comment here explaining why this flag was added and intended purpose.

Comment (Yasasr1) [reply components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java]: Updated with [36b998d](https://github.com/wso2/carbon-identity-framework/pull/5700/commits/36b998d3cb7980a87c8d3fabe1bfb956dec6124e)

File: components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java [hunk 2]
```diff
@@ -212,6 +215,12 @@ public ServiceURLBuilder addFragmentParameter(String key, String value) {
         return this;
     }
 
+    @Override
+    public ServiceURLBuilder setSkipDomainBranding(boolean skipDomainBranding) {
```
Comment (sadilchamishka) [inline components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java:219]: Add new line.

Comment (Yasasr1) [reply components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java:219]: Updated with [f6a868b](https://github.com/wso2/carbon-identity-framework/pull/5700/commits/f6a868bb49d4d214f2618995ee68fac9f7499ada)

**Review by sadilchamishka — APPROVED** [review]

**Pull request merged**